lewis/scripts/control.py

```diff
@@ -41,16 +41,20 @@ def show_api(remote, object_name):
     methods = []
 
     for member_name in dir(obj):
-        if not member_name[0] in ('_', ':'):
-            member = getattr(obj, member_name)
-
-            if callable(member):
-                methods.append(member_name)
+        if member_name[0] not in ('_', ':') and member_name not in dir(type(obj)):
+            methods.append(member_name)
```

Comment:
Won't this classify...
```python
class Foobar:
    hello = 'world'
```
...as a method since it's in `dir(type(obj))`? Or is this the client side "fake" `obj` that will look different?